panther_analysis_tool/validation.py

```diff
@@ -42,7 +42,8 @@ def contains_invalid_table_names(
         except Exception:  # pylint: disable=broad-except
             # Intentionally broad exception catch:
             # We want to fall back on original behavior if this third-party parser cannot tell us the table names
-            logging.info("Failed to parse query for scheduled query %s", analysis_id)
+            logging.info("Failed to parse query %s. Skipping table name validation", analysis_id)
```

Comment:
nit: imo, f-strings could be slightly more readable on the log lines in this file

Reply:
I agree. I can switch it

Reply:
Ah yes I forgot. The linter gets mad if you user f strings in the logging functions